api/src/routers/post.router.ts

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+import { getAllPosts } from '../services/post.service';
+
+const postRouter = express.Router();
+
+postRouter.get('/', getAllPosts);
+postRouter.get('/error', (req, res, next) => {
```

Comment:
This should be for testing purpose, do not forget to remove it before merging into master (and deliver to 'end-users' 😄 )